File: MED-CLOUD/med-ext/med-ext-orca/src/main/java/nta/med/orca/gw/api/contracts/message/MedicationInfo.java
package nta.med.orca.gw.api.contracts.message;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;

/**
 * @author dainguyen.
 */
@JsonIgnoreProperties(ignoreUnknown = true)
public class MedicationInfo {

    private String medicationCode;
    private String medicationName;
    private String medicationNumber;
    private String medicationGenericFlg;

    @JacksonXmlProperty(localName = "Medication_Code")
    public String getMedicationCode() {
        return medicationCode;
    }

    public void setMedicationCode(String medicationCode) {
        this.medicationCode = medicationCode;
    }

    @JacksonXmlProperty(localName = "Medication_Name")
    public String getMedicationName() {
        return medicationName;
    }

    public void setMedicationName(String medicationName) {
        this.medicationName = medicationName;
    }

    @JacksonXmlProperty(localName = "Medication_Number")
    public String getMedicationNumber() {
        return medicationNumber;
    }

    public void setMedicationNumber(String medicationNumber) {
        this.medicationNumber = medicationNumber;
    }

    @JacksonXmlProperty(localName = "Medication_Generic_Flg")
    public String getMedicationGenericFlg() {
        return medicationGenericFlg;
    }

    public void setMedicationGenericFlg(String medicationGenericFlg) {
        this.medicationGenericFlg = medicationGenericFlg;
    }
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Nuro/MedicationInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Nuro
{
    public class MedicationInfo
    {
        private String _medicationCode;
        private String _medicationName;
        private String _medicationNumber;
        private String _medicationGenericFlg;

        public String MedicationCode
        {
            get { return this._medicationCode; }
            set { this._medicationCode = value; }
        }

        public String MedicationName
        {
            get { return this._medicationName; }
            set { this._medicationName = value; }
        }

        public String MedicationNumber
        {
            get { return this._medicationNumber; }
            set { this._medicationNumber = value; }
        }

        public String MedicationGenericFlg
        {
            get { return this._medicationGenericFlg; }
            set { this._medicationGenericFlg = value; }
        }

        public MedicationInfo() { }

        public MedicationInfo(String medicationCode, String medicationName, String medicationNumber, String medicationGenericFlg)
        {
            this._medicationCode = medicationCode;
            this._medicationName = medicationName;
            this._medicationNumber = medicationNumber;
            this._medicationGenericFlg = medicationGenericFlg;
        }

    }
}